Dockerfile

```diff
@@ -94,62 +104,107 @@ ARG ACTIONS_RUNTIME_TOKEN
 ARG SCCACHE_MEMCACHED
 
 # Disable incremental buildings, not supported by sccache
-ENV CARGO_INCREMENTAL=false
+ARG CARGO_INCREMENTAL=false
 
 # Select whether we want dev or release
-ARG CARGO_BUILD_PROFILE=dev
+ARG CARGO_BUILD_PROFILE=debug
```

Comment:
It seems there is no profile `debug`. We should use `dev` instead and fix scripts where we expect `CARGO_BUILD_PROFILE == debug`